### ISSUE TITLE
AttributeError : 'Pack' object has no attribute 'set'

### ISSUE THREAD

**Issue by MisterY** [pybee/toga#622]
There's an issue with the similar error, #588, but that one refers to Linux.

## Expected Behavior

The demo should run on Windows.

## Current Behavior

Running the Hello World demo with `python -m hello` results in 

```
(venv) C:\src\python-gui\bee-toga>python -m hello

Unhandled Exception: Python.Runtime.PythonException: AttributeError : 'Pack' object has no attribute 'set'
   at Python.Runtime.Dispatcher.Dispatch(ArrayList args)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()

(venv) C:\src\python-gui\bee-toga>
```

## Steps to reproduce

Making the very first steps on Windows platform with the same code that worked yesterday on Linux. Following the [tutorial](https://toga.readthedocs.io/en/latest/tutorial/tutorial-0.html).

1. Create a virtual environment
2. Copy the code from [tutorial](https://toga.readthedocs.io/en/latest/tutorial/tutorial-0.html) into hello.py
3. Run `python -m hello`

## Your Environment

* Python Version 3.7.0

* Toga versions:
  * toga, toga-core, toga-winforms 0.3.0.dev11

* pythonnet 2.4.0.dev0 (for python 3.7)

* Operating System and Version (select from the following and list the specific version number; if your OS is not listed, list that as well)

  - [ ] macOS - version: 
  - [ ] Linux - distro:   - version:
  - [x] Windows - version: Windows 10, version 1709, build 16299.611
  - [ ] Other - name:  - version:

* Toga Target (the type of app you are trying to generate)
 
  - [ ] android
  - [ ] cocoa
  - [ ] django 
  - [ ] gtk
  - [ ] iOS
  - [ ] tvOS
  - [ ] watchOS
  - [ ] winforms 
  - [ ] win32
  - [x] Other (please specify): did not specify the version. Whatever is the default on Windows. I guess .Net?

**Comment by freakboy3742** [comment]
As with #588, the issue appears to be version incompatibility - at a guess, i'd say you're running an old version of the tutorial code against a current version Toga. Pack *doesn't* have an attribute "set" - however, in a previous version, it did.

You've provided a link to tutorial code - but I strongly suspect that's not actually the tutorial that you've used. The tutorial you've linked doesn't make any `set` calls.

**Comment by MisterY** [comment]
Thanks for your time and assistance with this issue. Unlike with openSuse, where core library was 0.3.0 dev but the others were 0.2.15, on Windows all the libraries are the same version, fortunately.

![image](https://user-images.githubusercontent.com/462445/47779831-5a511c80-dcfa-11e8-9ab1-3277b5f82903.png)

The hint about the tutorial version was a good one. I'm not sure how and why (sounds familiar?) but I've manually switched between 0.2.15 and the latest now and, true, the code I see in the example is different.

And, tadaaam, the Hello World window opens!

In retrospect, my assumption is that I read some of the previous issues and used the 0.2.15 tutorial code because, initially, I was doing this on Linux. There was a mixture of library versions and I've experienced strange [issues](https://github.com/pybee/toga/issues/621) until I've synchronized all versions to 0.2.15. Guess I also updated the tutorial code, since I remember the Hello World worked in that environment.

Now, on Windows, I get 0.3 dev versions installed automatically. I guess I should try forcing the installation of 0.3.0 on Linux, as well, and continue from there.
I'm looking to share code between Linux, Windows, and Android. Will report how it goes.

Thanks again and I'm looking forward to exploring more. This is currently my favourite approach in multi-platform app development.
